classes/NewsroomHelper.inc

```diff
@@ -214,17 +206,19 @@ class NewsroomHelper {
    *   Field data.
    */
   public static function getNodesAssociatedToTopics($topic_ids) {
-    $topic_ids = is_array($topic_ids) ? $topic_ids : array($topic_ids);
+    $topic_ids = is_array($topic_ids) ? $topic_ids : [$topic_ids];
     $cache_key = 'get_topic_entities' . md5(implode('', $topic_ids));
+    $data = NULL;
     if ($cache = cache_get($cache_key, NEXTEUROPA_NEWSROOM_CACHE_TABLE)) {
       $data = $cache->data;
     }
     else {
-      $query = db_select('field_data_field_newsroom_associated_topic', 'f')
-        ->fields('f')
-        ->condition('f.field_newsroom_associated_topic_tid', $topic_ids)
-        ->execute();
-      $data = $query->fetchAll();
+      $query = db_select('field_data_field_newsroom_associated_topic', 'f');
+      $query->fields('f');
+      $query->leftJoin('node', 'n', 'f.entity_id = n.nid');
```

Comment:
seems it didn't happen, for now it's ok, but please in the next release analyse this proposal.